### ISSUE TITLE
Far fetched suggestion: button to toggle a sheet to almost full page

### ISSUE THREAD

**Issue by oscar6echo** [QuantStack/ipysheet#6]
A sheet sometimes need be quite large - if the input is complex, or maybe also if the output is big.
So a button to temporarily expand the sheet to full page (or almost) proportions is very convenient in this context.

This suggestion comes from [qgrid](https://github.com/quantopian/qgrid).  
And I got good feedback from this feature in particular.

Food for thought.

**Comment by maartenbreddels** [comment]
Yes, that would be nice. I think the best way to attack it is to get this in ipywidgets, maybe it should be a (default disabled) feature of VBox/HBox.
Although there are multiple way to see fullscreen, like full fullscreen not seeing the browser any more, or just taking up the whole browser tab. I guess there are also extensions (from bloomberg?) that will fullscreen the output of a cell.
cc @jasongrout @SylvainCorlay
